setup.cfg

```diff
@@ -58,9 +59,10 @@ install_requires =
 	scikit-learn
 	scikit-misc
 	louvain
+    leidenalg
 	umap-learn
 	pydot
-	python-igraph
+	igraph
```

Comment:
is `igraph` now the official install? I can't find the old `python-igraph` as separate anymore, so i guess so?

Reply:
The python-igraph package page indicates that it is deprecated and igraph should be used instead. But I think leidenalg also forces same igraph adn python-igraph versions

Reply:
okay, thx.